src/jc_test.h

```diff
@@ -976,13 +980,35 @@ template<template <typename T> class BaseClass> struct jc_test_template_sel {
                 JC_TEST_MAKE_NAME2(testfixture,Types)>::register_test(#testfixture, #testfn, 0);            \
     template<typename T> void JC_TEST_MAKE_CLASS_NAME(testfixture,testfn)<T>::TestBody()
 
+#define JC_TEST_CAT_IMPL(s1, s2) s1##s2
+#define JC_TEST_CAT(s1, s2) JC_TEST_CAT_IMPL(s1, s2)
```

Comment:
THis is already implemented with `JC_TEST_MAKE_NAME2`. Please use that instead.

Reply:
Yes, I should have done that.